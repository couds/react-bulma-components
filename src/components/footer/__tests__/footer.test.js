import React from 'react';
import { render } from 'react-testing-library';
import { Footer } from '..';

describe('Footer component', () => {
  it('should render', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
