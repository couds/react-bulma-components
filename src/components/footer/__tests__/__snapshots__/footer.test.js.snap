// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render 1`] = `
<DocumentFragment>
  <div
    class="footer"
  />
</DocumentFragment>
`;
